python-lib/common.py

```diff
@@ -10,3 +10,12 @@ def get_lnt_path(path):
     elts = path.split('/')
     elts = [e for e in elts if len(e) > 0]
     return '/' + '/'.join(elts)
+
+
+def get_from_json_path(path, json_object):
```

Comment:
TO comment: given a array of nested fields, navigate to the terminal field and extract the value.
I think you should rename edge by node.